extension/src/experiments/workspace.ts

```diff
@@ -329,6 +329,15 @@ export class WorkspaceExperiments extends BaseWorkspaceWebviews<
     )
   }
 
+  public getHasData(dvcRoots: string[]) {
```

Comment:
This shouldn't need `dvcRoots` as it has the `repositories` object available. Could use this:
```
  public hasData() {
    return Object.values(this.repositories).some(repository =>
      repository.hasData()
    )
  }

```
or 

```
    for (const repository of Object.values(this.repositories)) {
      if (repository.getHasData()) {
        return true
      }
    }
    return false
  }
```